packages/govuk-frontend/src/govuk/init.jsdom.test.mjs

```diff
@@ -143,6 +175,42 @@ describe('initAll', () => {
       })
     )
   })
+
+  it('executes onError callback on component create if specified', () => {
+    document.body.classList.add('govuk-frontend-supported')
+    document.body.innerHTML = '<div data-module="govuk-accordion"></div>'
+
+    jest.mocked(GOVUKFrontend.Accordion).mockImplementation(() => {
+      throw new Error('Error thrown from accordion')
+    })
+
+    const errorCallback = jest.fn((error, context) => {
+      console.log(error)
+      console.log(context)
```

Comment:
One for a separate PR, but this makes me think that one thing we didn't test is that `createAll` (and `initAll`) no longer log to the console if an `onError` is provided. Would be great if we had a way to test this alongside the fact that the callback is called.

If those two `console.log` calls are only here to support the two `toHaveBeenCalledWith` later on, it seems [`toHaveBeenCalledWith` supports multiple arguments](https://jestjs.io/docs/expect#tohavebeenlastcalledwitharg1-arg2-) so those later calls might be doable as one:

```js
expect(errorCallback).toHaveBeenCalledWith(
  expect.any(Error),
  expect.objectContaining({
    // Whatever we expect
  })
)
```

Which would then allow us to check that the function no longer logged with

```js
expect(global.console.log).not.toHaveBeenCalled()
```

Reply:
I can consolidate the console.log check! I mean in theory it doesn't need to do any outputting at all. We can just check to see the error callback was called at all and then check to see if there's no error logged (the default behaviour).

Reply:
Created https://github.com/alphagov/govuk-frontend/issues/5303 to track the update of the tests.